corehq/project_limits/tests/test_system_limit.py

```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+
+from corehq.project_limits.models import SystemLimit
+
+
+class TestSystemLimitMethods(TestCase):
+
+    def test_for_key_returns_none_if_no_limit_set(self):
+        self.assertIsNone(SystemLimit.for_key("imaginary_limit"))
+
+    def test_for_key_returns_general_limit(self):
+        SystemLimit.objects.create(key="general_limit", limit=10)  # domain defaults to blank
+        self.assertEqual(SystemLimit.for_key("general_limit"), 10)
```

Comment:
Maybe it's overkill, but would it make sense to add a test similar to this one except with
```py
self.assertEqual(SystemLimit.for_key("general_limit", domain="no_match"), 10)
```
Or maybe just add that extra assertion in this test?

Reply:
[8bbb387](https://github.com/dimagi/commcare-hq/pull/35781/commits/8bbb387a888396cd3858e34bb4ba5c040ff48a4a)